hokusai/commands/env.py

```diff
@@ -3,6 +3,7 @@
 from hokusai.services.configmap import ConfigMap
 from hokusai.lib.common import print_green, print_smart
 from hokusai.lib.exceptions import HokusaiError
+from hokusai.lib.log_configmap_changes import log_configmap_changes
```

Comment:
Or should this be a `service` under `hokusai/services`?

Reply:
Services is somewhat motivated by integrations with other apis / programs such as Docker, Kubectl, etc.... though YamlSpec may be an exception it is a reference to a templating engine - yeah I think this may be a good case for putting it under services.

Reply:
okay.